src/tools/dev/scripts/visit-build-open

```diff
@@ -397,51 +397,47 @@ fi
 rm -f poodle
 cat <<EOF > poodle
 #!/bin/sh
+set -e
 XLOCALEDIR=/usr/lib/X11/locale
 PATH=/usr/bin/X11:\$PATH
-if test ! -d /usr/tmp/$user ; then
-   mkdir /usr/tmp/$user
-fi
+PATH=/usr/workspace/wsa/visit/visit/thirdparty_shared/3.4.0/toss4/cmake/3.24.3/linux-x86_64_gcc-10.3/bin/:\$PATH
```

Comment:
`/usr/workspace/wsa/visit...`

do we still need `wsa`?

Reply:
good catch, I dropped wsa. 

Still works in this case, but as we have seen may not work on future systems so we should move to new name.